config/crds/clusterlink.net_imports.yaml

```diff
@@ -36,8 +36,9 @@ spec:
             description: Spec represents the attributes of the imported service.
             properties:
               lbScheme:
-                description: LBScheme is the load-balancing scheme to use (e.g., Random,
-                  Static, ECMP)
+                default: ecmp
```

Comment:
what is the difference between `ecmp` and `random` modes?
I think `ecmp`  is perhaps too "low-level-network-stuff" for most users and we ought to select a simpler name (e.g., `hash`?)

Reply:
ECMP is implemented as a round-robin I will open an issue for that. @zivnevo

Reply:
May I suggest that we have one PR that sets the default and renames it to round-robin (or `rr`) across all files and samples? I counted 9 affected files.